From eabc74a2e9e42cc6b7fdbc7630a24ca8d656c7b4 Mon Sep 17 00:00:00 2001
From: Martin Kroeker <martin@ruby.chemie.uni-freiburg.de>
Date: Wed, 22 Feb 2017 09:04:33 +0100
Subject: [PATCH 1/3] Fix missing comma on continued line

EXTERNAL declaration of subroutines missed a comma before the continuation line,
causing a strange run-together name to appear in the object when compiled with ifort.
---
 SRC/chbev_2stage.f | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SRC/chbev_2stage.f b/SRC/chbev_2stage.f
index ff11506..5ced8c9 100644
--- a/SRC/chbev_2stage.f
+++ b/SRC/chbev_2stage.f
@@ -247,7 +247,7 @@
       EXTERNAL           LSAME, SLAMCH, CLANHB, ILAENV
 *     ..
 *     .. External Subroutines ..
-      EXTERNAL           SSCAL, SSTERF, XERBLA, CLASCL, CSTEQR
+      EXTERNAL           SSCAL, SSTERF, XERBLA, CLASCL, CSTEQR,
      $                   CHETRD_2STAGE
 *     ..
 *     .. Intrinsic Functions ..
-- 
2.8.1.windows.1

